Lib/test/test_concurrent_futures/test_process_pool.py

```diff
@@ -218,6 +228,81 @@ def mock_start_new_thread(func, *args, **kwargs):
                     list(executor.map(mul, [(2, 3)] * 10))
             executor.shutdown()
 
+    def test_process_pool_executor_terminate_kill_workers(self):
+        for function_name in ('terminate_workers', 'kill_workers'):
+            with self.subTest(function_name=function_name):
```

Comment:
```suggestion
            with self.subTest(function_name):
```

And ditto for the other places